src/lsp/cobol_indent/indenter.ml

```diff
@@ -62,8 +62,8 @@ let indenter ~source_format (str:string) (rdl:indent_record list) range =
   String.concat "\n" strl
 
 (*indent a range of file, with the default indent_config*)
-let indent_range' ~source_format ~range ~file =
-  let file_content = Ez_file.V1.EzFile.read_file file in
+let indent_range' ~source_format ~range input =
```

Comment:
I don't understand this change: you are replacing a name that is very specific `file` with a generic name `input`, where you completely lose the fact that it MUST be a file.

Reply:
Indeed, that was remains of some quick attempt to relax the limitation mentioned in the comment below. Now that's been reverted.